wbooks/src/app/components/button/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text, View, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+import { getButtonStyle } from './utils';
+
+export interface Props {
+  text: string;
+  stylesButton: string;
+}
+
+function Button({ text, stylesButton }: Props) {
+  const { styleText, styleBackground } = getButtonStyle(stylesButton);
+  return (
+    <TouchableOpacity>
+      <View style={[styles.btnDefault, styleBackground]}>
+        <Text style={[styles.btnTextDefault, styleText]}> {text} </Text>
+      </View>
```

Comment:
the view is unnecessary u can send the styles to the touchableOpacity.

Reply:
The View is removed and the styles are passed to the TouchableOpacity